### PR TITLE
[INT-971] Change the name in menus, update logo, 7 days delay

### PR DESCRIPTION
**Pull request by rokeras**
https://omnisend.atlassian.net/browse/INT-971

### CODE REVIEW

File: omnisend/class-omnisend-core-bootstrap.php [hunk 1]
```diff
@@ -182,17 +176,22 @@ public static function cron_schedules( $schedules ) {
...
-		$is_store_connected = Options::is_store_connected();
-		$visit_count        = Options::get_landing_page_visit_count();
+		$landing_page_visited            = Options::is_landing_page_visited();
+		$landing_page_notification_state = Options::get_landing_page_notification_state();
+		$landing_page_last_visit_time    = Options::get_landing_page_last_visit_time();
 
-		$store_connected_before_visit_count_option = $is_store_connected && $visit_count === 0;
-		if ( $store_connected_before_visit_count_option ) {
+		if ( $landing_page_visited && ( $landing_page_notification_state === NOTIFICATION_NOT_SHOWN || $landing_page_last_visit_time === 0 ) ) {
 			Options::set_landing_page_visited();
```
Comment (arnas) [inline omnisend/class-omnisend-core-bootstrap.php:184]: this seems illogical, if landing_page_visited then call set_landing_page_visited

File: omnisend/includes/Internal/class-options.php [hunk 2]
```diff
@@ -58,23 +63,45 @@ public static function is_connected(): bool {
...
+			$notification_state = NOTIFICATION_DELAYED;
+		} elseif ( $notification_state === NOTIFICATION_DELAYED && ( $current_time - $last_visit_time ) > self::get_notification_delay_time() ) {
+			$notification_state = NOTIFICATION_DISABLED;
+		}
+
+		update_option( self::OPTION_LANDING_PAGE_NOTIFICATION_STATE, $notification_state );
+		update_option( self::OPTION_LANDING_PAGE_VISIT_LAST_TIME, $current_time );
+		update_option( self::OPTION_LANDING_PAGE_VISITED, true );
+
+		return true;
```
Comment (arnas) [inline omnisend/includes/Internal/class-options.php:100]: why return true?

File: omnisend/includes/Internal/class-options.php [hunk 3]
```diff
@@ -58,23 +63,45 @@ public static function is_connected(): bool {
...
 
 		return is_numeric( $last_visit_time ) ? intval( $last_visit_time ) : 0;
 	}
 
+	public static function get_landing_page_notification_state(): string {
+		$notification_state = get_option( self::OPTION_LANDING_PAGE_NOTIFICATION_STATE );
+		return is_string( $notification_state ) ? $notification_state : NOTIFICATION_NOT_SHOWN;
+	}
+
+	public static function set_landing_page_notification_state( $state ): bool {
```
Comment (arnas) [inline omnisend/includes/Internal/class-options.php:77]: not used?

File: omnisend/class-omnisend-core-bootstrap.php (hunk 1 above)
Comment (rokeras) [reply omnisend/class-omnisend-core-bootstrap.php:184]: read whole condition

File: omnisend/includes/Internal/class-options.php (hunk 2 above)
Comment (rokeras) [reply omnisend/includes/Internal/class-options.php]: changed to void

File: omnisend/includes/Internal/class-options.php (hunk 3 above)
Comment (rokeras) [reply omnisend/includes/Internal/class-options.php]: removed

**Review by nerijuszaniauskas — DISMISSED** [review]

**Review by arnas — APPROVED** [review]

**Pull request merged**